### ISSUE TITLE
 val_org and val_gt ? 

### ISSUE THREAD

**Issue by Ftord** [kwwcv/iaanet#2]
where is datasets of val_org and val_gt ?

**Comment by Ftord** [comment]
I don't know how to look forward these images and masks of val.

**Comment by kwwcv** [comment]
MDvsFA dose not provide val set.
YOU can divide the training set into training and val sets. Or use data from other datasets for validation. OR directly use the test set of MDvsFA for validation and test on other datasets like sirst.
